src/sampling.jl

```diff
@@ -504,9 +508,11 @@ naive_wsample_norep!(a::AbstractArray, wv::WeightVec, x::AbstractArray) =
 #     URL http://www.sciencedirect.com/science/article/pii/S002001900500298X
 #
 # Instead of keys u^(1/w) where u = random(0,1) keys w/v where v = randexp(1) are used.
-function efraimidis_a_wsample_norep!(rng::AbstractRNG, a::AbstractArray, wv::WeightVec, x::AbstractArray)
+function efraimidis_a_wsample_norep!(rng::AbstractRNG, a::AbstractArray,
+                                     wv::WeightVec, x::AbstractArray)
     n = length(a)
-    length(wv) == n || throw(DimensionMismatch("a and wv must be of same length (got $n and $(length(wv)))."))
+    length(wv) == n || throw(DimensionMismatch(
+	    "a and wv must be of same length (got $n and $(length(wv)))."))
```

Comment:
Sorry for being dense. `throw(DimensionMismatch("a and wv must be of same length (got $n and $(length(wv)))."))` is 87 characters. Whether the current line is broken after `||` or a `if` block is used, that has to be preceded by 8 characters of indentation. 8 + 87 > 92, so the `throw` call must be broken across a line, right? Where is the natural place? (I'm happy to do whatever you tell me, not sure what the fix is here. Or you can push to the PR, right?)

Or define the string on a different line?
```
if length(wv) != n
    s = "a and wv must be of same length (got $n and $(length(wv)))."
    throw(DimensionMismatch(s))
end
```